BlueprintUICommonControls/Tests/Sources/PerformancePlayground.swift

```diff
@@ -24,7 +24,7 @@ class PerformancePlayground: XCTestCase {
 
     override func invokeTest() {
         // Uncomment this line to run performance metrics, eg in Instruments.app.
-        super.invokeTest()
+        // super.invokeTest()
```

Comment:
I think this was previously committed accidentally. Disabled again to make the tests run faster.